examples/dune

```diff
@@ -9,3 +9,7 @@
  (wrapped false)
  (modules RFC2616)
  (libraries angstrom))
+
+(alias
+ (name examples)
+ (deps RFC7159.cmxa RFC2616.cmxa))
```

Comment:
That's not right I think -- it'll only work on native code architectures. `dune build @all` would try to build them, so perhaps use that instead of the examples alias?